Unclear error message

I have a simple helmfile and the error `panic: runtime error: index out of range [0] with length 0` is not very expressive as I don't know where there is something wrong 

```yaml
repositories:
  - name: loki
    url: https://grafana.github.io/loki/charts

releases:
  - name: loki
    namespace: loki
    chart:
    # DB host, port, and connection opts for the environment
    values:
      # - values.yaml
```


```sh
helmfile sync
Adding repo loki https://grafana.github.io/loki/charts
"loki" has been added to your repositories

Updating repo
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "loki" chart repository
Update Complete. ⎈ Happy Helming!⎈ 

panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/roboll/helmfile/pkg/state.normalizeChart(0x1f46394, 0x1, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/helmfile-20191121-42300-iet6cw/helmfile-0.93.1/pkg/state/util.go:48 +0xf5
github.com/roboll/helmfile/pkg/state.(*HelmState).BuildDeps(0xc00049e000, 0x221ef40, 0xc00042e000, 0xc00042e000, 0x0, 0x0)
	/private/tmp/helmfile-20191121-42300-iet6cw/helmfile-0.93.1/pkg/state/state.go:1407 +0x1cc
github.com/roboll/helmfile/pkg/app.(*App).sync(0xc000430000, 0xc000368420, 0x2211fc0, 0xc00006a020, 0x30, 0x1e65360, 0x1, 0xc000368420)
	/private/tmp/helmfile-20191121-42300-iet6cw/helmfile-0.93.1/pkg/app/app.go:923 +0x1986
github.com/roboll/helmfile/pkg/app.(*App).Sync.func1(0xc000368420, 0xc000368420, 0x0, 0x0, 0x0)
	/private/tmp/helmfile-20191121-42300-iet6cw/helmfile-0.93.1/pkg/app/app.go:142 +0x46
github.com/roboll/helmfile/pkg/app.(*App).ForEachState.func1(0xc00049e000, 0x221ef40, 0xc00042e000, 0x0, 0xc000474e00, 0x1, 0xc000474e00)
	/private/tmp/helmfile-20191121-42300-iet6cw/helmfile-0.93.1/pkg/app/app.go:419 +0x90
github.com/roboll/helmfile/pkg/app.(*App).visitStates.func1(0x1f50eea, 0xd, 0xc00003e1b0, 0x2e, 0x0, 0x0)

```

--- a/pkg/app/desired_state_file_loader.go
+++ b/pkg/app/desired_state_file_loader.go
@@ -207,6 +207,12 @@ func (ld *desiredStateLoader) renderAndLoad(env, overrodeEnv *environment.Enviro
 			return nil, err
 		}
 
+		for i, r := range currentState.Releases {
+			if r.Chart == "" {
+				return nil, fmt.Errorf("error during %s parsing: encountered empty chart while reading release %q at index %d", id, r.Name, i)
+			}
+		}
+
 		if finalState == nil {
 			finalState = currentState
 		} else {

